example/spec/ingestGranule/IngestGranuleSuccessSpec.js

```diff
@@ -48,15 +51,39 @@ describe('The S3 Ingest Granules workflow', () => {
 
   describe('the PostToCmr Lambda', () => {
     let lambdaOutput;
+    let files;
+    const existCheck = [];
 
     beforeAll(async () => {
       lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, 'PostToCmr');
+      files = lambdaOutput.payload.granules[0].files;
+      existCheck[0] = await s3ObjectExists({ Bucket: files[0].bucket, Key: files[0].filepath });
+      existCheck[1] = await s3ObjectExists({ Bucket: files[1].bucket, Key: files[1].filepath });
+      existCheck[2] = await s3ObjectExists({ Bucket: files[2].bucket, Key: files[2].filepath });
+    });
+
+    afterAll(async () => {
+      await s3().deleteObject({ Bucket: files[0].bucket, Key: files[0].name }).promise();
+      await s3().deleteObject({ Bucket: files[1].bucket, Key: files[1].name }).promise();
+      await s3().deleteObject({ Bucket: files[2].bucket, Key: files[2].name }).promise();
     });
 
     it('has expected payload', () => {
       const granule = lambdaOutput.payload.granules[0];
       expect(granule.published).toBe(true);
       expect(granule.cmrLink.startsWith('https://cmr.uat.earthdata.nasa.gov/search/granules.json?concept_id=')).toBe(true);
     });
+
+    it('files move to correct location', () => {
```

Comment:
Change this description to something more in line with the acceptance criteria like "moves files to the bucket folder based on metadata". Patrick had sent something in the chat about that last week.